### PR TITLE
Add to contributing guidelines

### PR DESCRIPTION
**Pull request by ziadsawalha**
To help clarify some rules of engagement around
PRs and code submission.

To see the updated document click [here](https://github.com/ziadsawalha/simpl/blob/contributing/CONTRIBUTING.md).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -107,3 +107,201 @@ The following guidelines aim to solve this issue:
...
+* **Major Changes** that you wish to contribute to the project should be discussed first in an issue so that we can better coordinate our efforts, prevent
+duplication of work, and help you to craft the change so that it is successfully accepted into the
+project.
+* **Small Changes** can be crafted and submitted to the [GitHub Repository][github] as a Pull Request.
+
+
+### Submitting a Pull Request
+Before you submit your pull request consider the following guidelines:
+
+* Search [GitHub](https://github.com/checkmate/chessboard/pulls) for an open or closed Pull Request
```
Comment (samstav) [inline CONTRIBUTING.md]: :point_up:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -107,3 +107,201 @@ The following guidelines aim to solve this issue:
...
+reference GitHub issues that this commit **Closes**.
+
+
+A detailed explanation can be found in this [document][commit-message-format].
+
+---
+Note: inspired by and lifted with gratitude from [AngularJS](http://www.angularjs.org)
+
+[checkmate-hackers]: https://groups.google.com/forum/#!forum/checkmate-hackers
+[github]: https://github.com/checkmate/chessboard
```
Comment (samstav) [inline CONTRIBUTING.md]: :point_up:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -107,3 +107,201 @@ The following guidelines aim to solve this issue:
...
+The footer should contain any information about **Breaking Changes** and is also the place to
+reference GitHub issues that this commit **Closes**.
+
+
+A detailed explanation can be found in this [document][commit-message-format].
+
+---
+Note: inspired by and lifted with gratitude from [AngularJS](http://www.angularjs.org)
+
+[checkmate-hackers]: https://groups.google.com/forum/#!forum/checkmate-hackers
```
Comment (samstav) [inline CONTRIBUTING.md]: Probably change this

**Comment by samstav** [comment]
This is great, @ziadsawalha. Thanks

File: CONTRIBUTING.md (hunk 1 above)
Comment (ziadsawalha) [inline CONTRIBUTING.md]: Should be fixed now. Thanks.

**Pull request merged**